sycl/test-e2e/OptionalKernelFeatures/is_compatible/is_compatible_spir64_x86_64.cpp

```diff
@@ -0,0 +1,6 @@
+// REQUIRES: gpu, level_zero, opencl
```

Comment:
Can you please also add tests for CUDA and HIP?

Reply:
[44d5c89](https://github.com/intel/llvm/pull/9769/commits/44d5c89980e3879df1784c952e554b61ff8f09a1)